code-challenges.test.js

```diff
@@ -36,14 +81,57 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 
 // b) Create the function that makes the test pass.
 
+const minMax = (array) => {
+  let min = array[0], max = array[0]
+
+for (let i = 1; i < array.length; i++) {
+  let value = array[i]
+  min = (value < min) ? value : min
+  max = (value > max) ? value : max
+}
+
+return [min, max]
+}
+// console.log(minMax(nums1));
+// console.log(minMax(nums2));
+
```

Comment:
Your function passes the test!!!

`describe("minMax", () => {`
`  it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {`

`    const nums1 = [3, 56, 90, -8, 0, 23, 6]`
`    // Expected output: [-8, 90]`
`    const nums2 = [109, 5, 9, 67, 8, 24]`
`    // Expected output: [5, 109]`

`    expect(minMax(nums1)).toEqual([-8, 90])`
`    expect(minMax(nums2)).toEqual([5, 109])`
`  })`
`})`